test/converters.test.ts

```diff
@@ -33,9 +56,36 @@ test("number converter", async () => {
   await check(converters.number, "19.14", 19.14);
   await check(converters.number, "19.", 19);
   await check(converters.number, "-3.14", -3.14);
+  await checkWithOptions(converters.number, "1234,56", 1234.56, {
```

Comment:
I'd like to see a test that verifies that "1,1.1" isn't acceptable. This because you have some tricky code where you normalize the last segment separately, and you exclude the last segment from the normal length == 3 check for that reason. But if you have a bug in there, this fragment would be accepted.  I see the code that should handle this, but I don't see a test that explicitly tries to trigger this problem, but perhaps I missed it.